src/hammer-vlsi/pcb/generic/defaults.yml

```diff
@@ -0,0 +1,21 @@
+# Configuration options and defaults for the generic PCB deliverable tool
+# The values specified in this file are the defaults.
+
+# A catch-all for generic PCB deliverables that needn't be in a separate plugin
+pcb.generic:
+
+  # Footprint type
+  # Currently only supports PADS-V9
```

Comment:
Is `PADS-V9` some type of format?

Reply:
Yes, but it's poorly documented. This pdf has it more or less, but there are some discrepancies I've noticed, specifically with the terminal entries: ftp://ftp.freecalypso.org/pub/CAD/PADS/pdfdocs/Plib_ASCII.pdf
It's in a comment buried in the plugin.

Reply:
Can we add that link to the file? I'd be happy to help edit but since it's a rebase-merge I'm not sure how you want the changes to fit into the commit history

Reply:
I got an email saying you force-pushed but did you do it twice? I don't see a difference now.

Reply:
I went ahead and updated it

Reply:
I edited it directly and then realized that you wanted to keep the history neat for rebase-merge so I removed my commit and added my comment here.

Reply:
Thanks!